Dynamic_Bar.lua

```diff
@@ -8,9 +8,7 @@ local Type = TMW.Classes.IconType:New("dynamicbar")
 Type.name = "Dynamic Bar"
 Type.desc = "A Bar that can be controlled by LUA scripts"
 Type.menuIcon = "Interface/Icons/inv_box_04"
-Type.unitType = "unitid"
```

Comment:
Your type won't have unit input and won't be checking units, so this isn't needed.